d3ploy/DemandFac.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+import scipy as sp
+import random
```

Comment:
random is from the standard library, so its import should go above the 3rd party imports